normandy/classifier/api.py

```diff
@@ -6,7 +6,8 @@
 
 
 class FetchBundle(views.APIView):
-
+    authentication_classes = []
+    permission_classes = []
```

Comment:
I think it's good to make a lack of CSRF the exception rather than the default. It's better to have a view that shouldn't have CSRF accidentally have it then to have the opposite.